### ISSUE TITLE
Uncaught TypeError: Designer is not a constructor

### ISSUE THREAD

**Issue by hand-dot** [pdfme/pdfme#413]
### Describe the bug

The build of @pdfme/ui was changed from webpack to vite in version 3.1.0. Probably due to this change, the Designer is not being built correctly in some cases.

There have been two reports of this error from users, both of which were built using vite.

- https://github.com/ryo-ma1010/MyLaravel/blob/master/resources/react/src/pdfme.tsx
- https://github.com/MRNafisiA/pdfme-bug-reproduce/blob/main/src/main.tsx

Both use React plugins and have .tsx extensions, but the code looks like vanilla ts.

In any case, we need to investigate the cause. It might also be a good idea to create several test environments to check which project setups it works with.

---

ref: https://github.com/pdfme/pdfme/discussions/409
ref: https://discord.com/channels/933602537808990268/933602537808990271/1199911773541777448

### To Reproduce

https://github.com/MRNafisiA/pdfme-bug-reproduce

### Expected behavior

The designer is ensuring that everything works correctly in the repository mentioned above.

### Your Environment

```markdown
- pdfme package(@pdfme/generator or @pdfme/ui): @pdfme/ui
- pdfme version: since 3.1.0
- Operating system: macOS
- Node.js version or Browser name & version: Node.js v20, Latest Google Chrome
```

### Your Error Log

```shell
pdfme.js:29 Uncaught TypeError: Designer is not a constructor
```

### Additional context

Workaround:
use v 3.0.1

![image](https://github.com/pdfme/pdfme/assets/24843808/ecf27fd3-fb6a-4a43-9e98-6d038feebd4c)

I guess v 3.1.0's change is affected.
https://github.com/pdfme/pdfme/releases/tag/3.1.0

![image](https://github.com/pdfme/pdfme/assets/24843808/77a7c9af-47a6-4a71-a96a-422f445429c7)

**Comment by hand-dot** [comment]
- working with vite and pdfme@3.0.1 : https://stackblitz.com/edit/vitejs-vite-m72yyd?file=package.json
- not working with vite and pdfme@^3.2.1: https://stackblitz.com/edit/vitejs-vite-jyrsj7?file=package.json

**Comment by hand-dot** [comment]
Fixed and release it as 3.2.2

After user check, I gonna close this issue.

**Comment by MRNafisiA** [comment]
I checked it and It is all fixed!
Thanks!
@hand-dot
